product_docs/docs/biganimal/release/using_cluster/extensions.mdx

```diff
@@ -1,14 +1,17 @@
 ---
-title: "Postgres extensions"
+title: "Managing Postgres extensions"
+redirects: 
+ - /biganimal/release/using_cluster/03_modifying_your_cluster/extensions
 ---
 
 BigAnimal supports many Postgres extensions. See [Postgres extensions available by deployment](/pg_extensions/) for the complete list.
 
+You need superuser privileges to install most extensions on BigAnimal. The table in [Postgres extensions available by deployment](/pg_extensions/) indicates whether an extension requires superuser privileges. If you are using your own cloud account, you can grant superuser privileges to edb_admin so that you can install these extensions on your cluster (see [superuser](/biganimal/latest/using_cluster/01_postgres_access/#superuser)). If you are using BigAnimal's account, you can't install the extensions that require superuser access. 
```

Comment:
This isn't the case, exactly, and as noted elsewhere I don't think we should be referring to that table - it's kinda confusing.

Reply:
```suggestion
Many Postgres extensions require superuser privileges to be installed. If you are using your own cloud account, you can grant superuser privileges to edb_admin so that you can install these extensions on your cluster (see [superuser](/biganimal/latest/using_cluster/01_postgres_access/#superuser)). If you are using BigAnimal's cloud account, where superuser in unavailable, some extensions can be installed with the pg_ba_admin role instead. See the table below for details.
```

Reply:
We should include here the list of BigAnimal-supported extensions that require superuser, and which are available to install on BAH.

Reply:
Is the list of extensions available on BAH those that don't require superuser?

Reply:
Reverting this change for now - until we decide what approach we are taking

Reply:
Committed this change with the reference to the extensions table added back in.